content/docs/user-guide/build/distributable-engine.md

```diff
@@ -0,0 +1,226 @@
+
+For multi-platform projects where project teams will be working from a distributable build, each distributable build should be placed in a distinct directory set through `CMAKE_INSTALL_PREFIX`.
+{{< /note >}}
+
+## Overview
+
+Creating a distributable build for project developers while keeping core engine developers in a separate path involves the following major steps:
+
+1. Building a local O3DE SDK with the `install` target
+1. Configuring your project to use the local SDK for builds
+1. Building the project
+1. Creating the distributable build of the engine
+1. Providing the distributable build to project developers
+
+The exact process involved in the last step is left up to your in-house development practices, although we recommend using source control with good blob support for your build distribution. For teams using open source solutions we recommend [git LFS](https://git-lfs.github.com/).
+
+When creating a new distributable build of the O3DE core tools, each step should be performed _every_ time a new build is created, to ensure that any changes to the engine which affect engine or project configuration are correctly picked up.
+
+## Create a local SDK
+
+The first step in creating a distributable build is generating local binaries from your engine source, used to build your project and create the final distributable engine build.
+
+### Build the `INSTALL` target
+
+Perform the following steps from the **O3DE source directory** (`C:\o3de-source` in this example):
+
+1.  Generate the toolchain project files using a unique `LY_VERSION_ENGINE_NAME` CMake cache setting. This value is the name of the engine used by the O3DE project manager and registration system.
+    
+    ```cmd
+    cmake -B build/windows_vs2019 -G "Visual Studio 16" --config profile -DLY_VERSION_ENGINE_NAME="MyO3DE"
+    ```
+
+    The **profile** configuration is recommended for distributed builds, as it provides additional logging and engine introspection capabilities useful during project development at minimal performance cost. **debug** builds are primarily useful during engine development.
+1.  Build the `INSTALL` target.
+    
+    ```cmd
+    cmake --build build/windows_vs2019 --target INSTALL --config profile
+    ```
+    
+    The binaries will be placed into the directory specified by the `CMAKE_INSTALL_PREFIX` CMake cache variable, or in an `install` subdirectory of the source code by default.
+
+Run the following steps from the install directory:
+
+1. Install the version of Python and modules required by the engine.
+   
+   ```cmd
+   python\get_python.bat
+   ```
+
+### Register the engine
+
+The next step in creating a distributable build of the engine tools is to register the locally installed engine. These steps should be re-run every time you create a new build of the `INSTALL` target.
```

Comment:
I'm not sure why you'd need to re-register the engine every time you build it to the install target, assuming the engine name and install path are kept the same. I'm curious as to the reason for stating this requirement.

Reply:
The engine only need to be registered if the `LY_VERSION_ENGINE_NAME` value was changed as that effects the engine name in the install layout